### PR TITLE
Fix other user's displayName being wrapped on the bubble message layout

### PR DESCRIPTION
**Pull request by luixxiul**
Fixes https://github.com/vector-im/element-web/issues/22004

This PR fixes the user name being wrapped on the bubble message layout.

Steps to confirm the fix:
1. Change the layout to bubble message layout
2. Create a test user
3. Set a long user name to the user
4. Create a test room
5. Invite the test user
6. Send a message from the test user
7. Open the room from your main account

|Before|After|
|---------|------|
|before | after|
|before|after|

Also it removes redundant rules, moving the cursor setting to `_DisambiguatedProfile.scss`

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: defect

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix other user's displayName being wrapped on the bubble message layout ([\#8456](https://github.com/matrix-org/matrix-react-sdk/pull/8456)). Fixes vector-im/element-web#22004. Contributed by @luixxiul.

### CODE REVIEW

**Comment by luixxiul** [comment]
There are cases where mxid should not be hidden, like this:

before

I'm going to look for another solution.

**Comment by luixxiul** [comment]
With those commits it can be displayed like below:

after

Long display names are truncated on the bubble layout, without hiding the mxid.

Design review would be appreciated.

**Review by kerryarchibald — COMMENTED** [review]
Requested design review for this change.

File: res/css/views/messages/_DisambiguatedProfile.scss [hunk 1]
```diff
@@ -18,16 +18,39 @@ limitations under the License.
...
         font-weight: 600;
         font-size: 1.1rem;
-        margin-left: 5px;
+        margin-inline-start: 5px;
         opacity: 0.5; // Match mx_TextualEvent
         color: $primary-content;
     }
 }
+
+.mx_EventTile[data-layout=bubble] {
```
Comment (luixxiul) [inline res/css/views/messages/_DisambiguatedProfile.scss:38]: This block is going to be moved to `_EventBubbleTile.scss`.

**Review by SimonBrandner — APPROVED** [review]
I wonder if something similar would work for the modern layout?

**Comment by luixxiul** [comment]
I don't think it would be impossible, though I am not sure if it is possible by editing CSS only.

**Pull request merged**